theatre/core/src/sheetObjects/SheetObjectTemplate.ts

```diff
@@ -103,8 +107,52 @@ export default class SheetObjectTemplate {
   }
 
   /**
-   * Filters through the sequenced tracks those tracks who are valid
-   * according to the object's prop types.
+   * Iterates through a tree of properties and returns the path and trackId if the trackId exists
+   *
+   * Returns an array.
+   */
+  getOrderedTrackIds({
```

Comment:
This should be just a function rather than a method. It doesn't use the rest of the class' API